### ISSUE TITLE
Consul-k8s Latest Does not have inject capabilities

### ISSUE THREAD

**Issue by dharnidhargundu** [hashicorp/consul-k8s#86]
Hi I am trying to Configure Consul-Connect in a Kubernetes Cluster and used the Docker Image from https://hub.docker.com/r/hashicorp/consul-k8s
The connect-inject-deployment container Fails with Following Error : 

Usage: consul-k8s [--version] [--help] <command> [<args>]

Available commands are:
    sync-catalog    Sync Kubernetes services and Consul services.
    version         Prints the version

Seems like the Latest Docker Image and Build Does not have inject related Features or Changes. Can someone please rebuild and publish New Image and Build with latest changes

**Comment by rboyer** [comment]
The latest version does have the `inject-connect` subcommand:
```
$ docker run --rm -it hashicorp/consul-k8s:0.7.0 consul-k8s inject-connect -h | head -n 5
Usage: consul-k8s inject-connect [options]

  Run the admission webhook server for injecting the Consul Connect
  proxy sidecar. The sidecar uses Envoy by default.
```

The `inject-connect` subcommand [is omitted from the help text](https://github.com/hashicorp/consul-k8s/blob/v0.7.0/commands.go#L34) as it is not something you would ordinarily run yourself without assistance from the consul helm chart.

**Comment by adilyse** [comment]
Hi @dgundu,

We've also just released the latest version (0.8.1) last week which includes Connect injection updates. Do you run into the same error with the latest version?

**Comment by lkysow** [comment]
I'm going to close this since the sync-catalog command does exist. If something isn't working please comment and I'll reopen.
